script/Arcueid/Day 1/01_00_ARC01_9.txt

```diff
@@ -207,28 +207,28 @@ What %{i}thing%{/i} is she talking about?
 -- あるでしょう。兄さんのズボンにも。若者の象徴のような汚らわしいアレが。
 You %{i}do%{/i} have one, don't you?
  In your pants.
- That filthy thing is the symbol of youth.
+ That filthy symbol of youth.
 }
 [sha:82bbb06ad057416b45d6fea1e86e47db58ebcf36]{
 -- Page 512, Offset 2461.
 -- それを隠さずに出しなさい、と言っているのです」
-I am telling you to stop hiding it and take it out."
+So I want you to stop hiding it and get it out already."
```

Comment:
Github won't let me comment on the entire section for some reason but I gotta say, I prefer the original version of this joke. 'Thing' is a good euphemism here, and the final command in the original is stronger.

Reply:
It sounds extremely JP > EN

Reply:
Well, the new version sounds kinda flaccid. Alternate suggestion
> But before that――Nii-san, please take out that %{i}thing%{/i}."
> 2457 no change
> My thing? What %{i}thing%{/i} [is she talking about]?!
> 2459 nc
> 2460 preference for original 
> Now quit stalling and show it to me already. / Now hurry up and take it out already.

Reply:
Nah, it sounds so much worse.

Reply:
Well, I'm afraid that the current suggestion feels like a significant downgrade. Happy to hear alternate suggestions but the delivery is a lot weaker overall.

Reply:
Definitely prefer 'thing' over 'it', especially because of 2458. "It? What 'it'?" just sounds really bad to me. Though I think changing it to something like "Thing? What 'thing'?" would be better.

I like Petri's new version for 2460.

For 2461, I'd just cut it down to 'Stop hiding it and get it out already.'

Reply:
So overall - 
> But before that――Nii-san, please take out that thing."
"What?"
Thing? What %{i}thing%{/i}?
"You know full well what I'm talking about.
You %{i}do%{/i} have one, don't you?
 In your pants.
 That filthy symbol of youth.
Stop hiding it and get it out already."
[...]
"Huh? I am telling you to take out your phone.

Reply:
God, it sounds so bad.